client/test/e2e/shared/components.js

```diff
@@ -32,3 +32,39 @@ exports.currencyInput = {
     return elm.getAttribute('ng-invalid');
   }
 };
+
+/**
+ * Object to help making end-to-end test,
+ * dateEditor.js.
+ * @public
+ */
+
+
+exports.dateEditor = {
```

Comment:
That's fine. Please include some example of this component being used, with an e2e test to demonstrate how to write a test for a module with it.